SAKPaaS/src/app/components/legal/legal-routing.module.ts

```diff
@@ -1,12 +1,38 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
 import { LegalComponent } from 'src/app/components/legal/legal.component';
+import { ImpressumComponent } from 'src/app/components/legal/impressum/impressum.component';
```

Comment:
Should be named `ImprintComponent` instead of `ImpressumComponent`. Please don't mix languages. 😀

Reply:
https://en.wikipedia.org/wiki/Impressum

So wie ich das sehe, ist das lateinisch und findet in beiden Sprachen Gebrauch. Impressum ist im Telemediengesetz festgeschrieben:
> Sobald deine Firma ihren Sitz in Deutschland hat oder sich dein Angebot direkt an den deutschen Markt wendet, bist du der Impressumspflicht unterlegen. Das gilt auch, wenn deine Website gar nicht in deutsch, sondern in einer anderen Sprache online ist.

Imprint ist im Bezug auf Websites eine falsche Übersetzung. Hab mal dir zuliebe bei Youtube & Facebook nachgeschaut, die übersetzen das auch jeweils mit `impressum`.

Reply:
> https://en.wikipedia.org/wiki/Impressum
> 
> So wie ich das sehe, ist das lateinisch und findet in beiden Sprachen Gebrauch. Impressum ist im Telemediengesetz festgeschrieben:
> 
> > Sobald deine Firma ihren Sitz in Deutschland hat oder sich dein Angebot direkt an den deutschen Markt wendet, bist du der Impressumspflicht unterlegen. Das gilt auch, wenn deine Website gar nicht in deutsch, sondern in einer anderen Sprache online ist.
> 
> Imprint ist im Bezug auf Websites eine falsche Übersetzung. Hab mal dir zuliebe bei Youtube & Facebook nachgeschaut, die übersetzen das auch jeweils mit `impressum`. @lepathie

Alright, got me convinced. Learning something new here. 🤣